src/types.tsx

```diff
@@ -354,9 +354,9 @@ export interface ScreenProps extends ViewProps {
    */
   statusBarHidden?: boolean;
   /**
-   * Sets the status bar color (similar to the `StatusBar` component). Requires enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file on iOS. Defaults to `auto`.
+   * Sets the status bar color (similar to the `StatusBar` component). Requires enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file on iOS. Defaults to `system`.
    */
-  statusBarStyle?: 'inverted' | 'auto' | 'light' | 'dark';
+  statusBarStyle?: 'auto' | 'system' | 'systemInverted' | 'light' | 'dark';
```

Comment:
So now `inverted` and `auto` will not be available at all? If so, I guess it will land in v4?

Reply:
Most likely. I was thinking if we want to keep `inverted` option that will be reverse of `auto` (or just keep it as a fallback option to `systemInverted`), but most likely we should introduce this change on v4, yeah.